docs/core/config/auth/passdb.md

```diff
@@ -442,6 +445,45 @@ a load spike of everybody getting logged in at exactly the same time.
 
 Do not perform any authentication, just store extra fields if user is found.
 
+##### `check_client_fp`
+
+Match client certificate or public key fingerprint.
+
+Requires configuring [[setting,ssl_peer_certificate_fingerprint_hash]], [[setting,auth_ssl_require_client_cert]],
+and [[setting,ssl_server_request_client_cert]] to require a client cert.
+
+This is intended to replace CA certificates with verifying client certificates using fingerprints, or to enforce
+that particular public key is being used. If you are using CA certificates,
+you can only fail authentication if the value is non-empty and does not match. If CA certificates are not used, then
+if no passdb matches the fingerprint, then authentication is failed.
+
+The failure is per-passdb, so next passdb can continue authenticating the user. The final result is checked after
+authentication, in which case either the certificate must be validated by CA, or by matching fingerprint in some passdb.
```

Comment:
I guess this could continue: If the certificate is not validated, the authentication fails with "Client didn't present valid SSL certificate".